README.md

```diff
@@ -8,6 +8,7 @@ The aims of the project are:
 * promote collabotation among _wizards_ and *witches*
 * make a *spell* database
 * form a **community**
+* annihilate every living unicorn
```

Comment:
Please change the word "annihilate" to "pet"

Reply:
No.